Makefile

```diff
@@ -105,7 +105,7 @@ uninstall: manifests kustomize ## Uninstall CRDs from the K8s cluster specified
 release-manifests: manifests kustomize
 	cd config/manager && $(KUSTOMIZE) edit set image controller=${IMG}
 	mkdir -p config/rendered/
-	$(KUSTOMIZE) build config/default > config/rendered/release.yaml
+	$(KUSTOMIZE) build config/overlays/helm > config/rendered/release.yaml	
```

Comment:
Doesn't adding this here now break the release.yaml inside of the artifacts on the repo?

Reply:
updated, added if statement from a `KUSTOMIZE_OVERLAY` variable, the helm build writes directly to `chart/templates/rendered.yaml` now